src/tzdata/version.jl

```diff
@@ -112,3 +112,16 @@ function active_archive()
     !isfile(archive) && error("Missing $version tzdata archive")
     return archive
 end
+
+if VERSION >= v"1.4"
+    function active_dir()
+        version = active_version()
+        archive = @artifact_str "tzdata_$version"
+        !isdir(archive) && error("Missing $version tzdata archive")
+        return archive
+    end
```

Comment:
Is this only used for testing?

Reply:
Yes. Should I delete it then?
I had it as Pkg friendly version of `active_archive`, which is also used only for testing.